tools/stress-cluster/services/Stress.Watcher/src/ChaosResources/Resource.cs

```diff
@@ -34,6 +35,45 @@ public class ChaosResourceSpec
     {
         [JsonProperty(PropertyName = "selector")]
         public ChaosSelector Selector { get; set; }
+
+        [JsonProperty(PropertyName = "networkChaos")]
+        public ChaosResourceSpec NetworkChaos { get; set; }
+        
+        [JsonProperty(PropertyName = "stressChaos")]
+        public ChaosResourceSpec StressChaos { get; set; }
+
+        [JsonProperty(PropertyName = "httpChaos")]
+        public ChaosResourceSpec HttpChaos { get; set; }
+
+        [JsonProperty(PropertyName = "ioChaos")]
+        public ChaosResourceSpec IoChaos { get; set; }
+
+        [JsonProperty(PropertyName = "kernelChaos")]
+        public ChaosResourceSpec KernelChaos { get; set; }
+
+        [JsonProperty(PropertyName = "timeChaos")]
+        public ChaosResourceSpec TimeChaos { get; set; }
+
+        [JsonProperty(PropertyName = "jvmChaos")]
+        public ChaosResourceSpec JvmChaos { get; set; }
+
+        public string GetTestInstance()
```

Comment:
This is a super nit, but I wonder if it would be better just to use a nullable string response type here, to differentiate between "testInstance label exists and is empty" vs. "no testInstance label." It would also simplify your below code a little bit.